packages/compose/src/hooks/use-copy-on-click/index.js

```diff
@@ -1,12 +1,7 @@
-/**
- * External dependencies
- */
-import Clipboard from 'clipboard';
```

Comment:
While the changes in this file are pretty convoluted, it is a deprecated hook after all. The changes in the other hook are a lot more palatable. :P I still think it's worth it to remove the dependency.

Reply:
I had the same train of thoughts, and came to the same conclusion — refactoring this deprecated hook allows us to remove the dependency from the repo :)